Scripts/DCS-BIOS/doc/Addresses.h

```diff
@@ -13617,6 +13620,7 @@
 #define F_5E_3_UTIL_PRESS_A 0x762E
 #define F_5E_3_VARIOMETER 0x7694, 0xFFFF, 0
 #define F_5E_3_VARIOMETER_A 0x7694
+#define F_5E_3_VHF_RADIO_A 0x772A
```

Comment:
Same here

Reply:
Yes, I mistook the two models (twice). So it got committed here.